src/status_im/contexts/wallet/bridge/select_asset/view.cljs

```diff
@@ -28,6 +29,7 @@
          :placeholder     (i18n/label :t/search-assets)}]
        [asset-list/view
         {:search-text    @search-text
+         :chain-ids      networks.config/bridge-supported-networks
```

Comment:
can add a function to `networks.core` that returns this config value. Would be good to isolate the usage of the config values by using functions, otherwise in case we change the implementation on how we decide whether a network is supported for bridging, we'll have more things to refactor.